doc-links.yml

```diff
@@ -122,11 +128,19 @@
     - title: Configuring Ambassador Edge Stack
       link: /reference/configuration
     - title: Configuration Format
-      items:     
+      items:
+        - title: Format Overview
+          link: /reference/config-format
+        - title: Kubernetes Service Annotations  
+          link: /reference/core/annotations/ 
```

Comment:
Um. You didn't re-add the `/reference/core/annotations` page.  Either add it, or remove this entry for it.

Reply:
added